### PR TITLE
Use braced sed substitution strings for YAMLS

### PR DESCRIPTION
**Pull request by jjguerrette**
### Description
This branch modifies all YAML substitution strings to be surrounded by double braces, i.e., `{{substitutionString}}` instead of `substitutionString`.  This modification makes it clearer to users and developers which variables are provided by the workflow and which ones are static.

A later PR will do the same for namelist and streams files.

Two additional unrelated changes in `PrepVariational.csh` cause (1) observation perturbations to be enabled for all EDA members, including the first member and (2)observation perturbations to be re-centered around zero. I.e., in commit ab2a490d9126c05ed83909f30c0447f9b532179d, 

image

### Issue closed

None

### Tests completed
 - [x] 3dvar_OIE120km_WarmStart
 - [x] 3denvar_OIE120km_WarmStart
 - [x] 3dvar_OIE120km_ColdStart
 - [x] 3denvar_O30kmIE60km_WarmStart
 - [x] eda_OIE120km_WarmStart

### CODE REVIEW

**Comment by jjguerrette** [comment]
One more commit coming, as I noticed a few situations that are missed in this branch.

File: config/ObsPlugs/variational/ObsAnchors.yaml [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
   Clouds: [Water, Ice, Rain, Snow, Graupel]
   linear obs operator:
     Absorbers: [H2O]
     Clouds: [Water, Ice, Rain, Snow, Graupel]
   obs options:
     <<: *CRTMObsOptions
 _get values: &GetValues
-  interpolation type: InterpolationType
+  interpolation type: {{InterpolationType}}
+_multi iteration filter: &multiIterationFilter
```
Comment (ibanos90) [inline config/ObsPlugs/variational/ObsAnchors.yaml:31]: Could you please add this in the individual yamls?

**Comment by ibanos90** [comment]
I ran the suggested tests and all tasks completed successfully except CleanHofx in the eda_OIE120km_WarmStart test. The task is failing specifically for member 4. Could you please check this?

**Review by ibanos90 — APPROVED** [review]

**Comment by jjguerrette** [comment]
@ibanos90, I do not mind waiting to merge this until after your real-time and QC PR's are initiated/merged.  Let me know your preference.

**Comment by ibanos90** [comment]
@jjguerrette, I think is fine if you merge it. I can use the "multiIterationFilter" in my QC PR's.

**Pull request merged**